packages/sdk/src/router/types.ts

```diff
@@ -38,6 +38,10 @@ export type GenericOrder =
   | {
       kind: "sudoswap";
       order: Sdk.Sudoswap.Order;
+    }
+  | {
```

Comment:
We'll also need to support it at the router level:
https://github.com/reservoirprotocol/core/blob/d2947e9e4f21f184851beec958daa99d0c6e7cdb/packages/sdk/src/router/router.ts#L406
https://github.com/reservoirprotocol/core/blob/d2947e9e4f21f184851beec958daa99d0c6e7cdb/packages/sdk/src/router/router.ts#L489